app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def append_info_to_payload(payload)
```

Comment:
@mrcasals we aren't forcing everyone to use lograge, I just want to add the `append_info_to_payload` to Decidim's `ApplicationController`, using this information with your logs is up to the developer.